src/PSRClassesInterface.jl

```diff
@@ -2,6 +2,7 @@ module PSRClassesInterface
 
 import Dates
 import JSON
+import JSONSchema
```

Comment:
where it is used?

Reply:
I tried out schema-based validation during some creation operations and it seemed very powerful to me.
A single `JSONSchema.validate` frees us from many `haskey` calls and deals natively with nested structures.
It is not currently active right now because it has a hard time dealing with unpredictable keys such as `"SomeKey(i,j)"`

Reply:
Lets remove the dependency.
I want to keep this as low weight as possible.
I can imagine a separate package PSRClassesSchemaValidation.
Please open a issue so we remember to go back to it